<launch>
  <arg name="human_heights" default="$(find hri_planner)/resources/video_processing/" />

  <node name="video_processor" pkg="hri_planner" type="video_processor" output="screen">
    <param name="camera_info_file" value="$(find hri_planner)/resources/camera/gopro1440.yml" />
    <param name="hat_tracker_config" value="$(find hri_planner)/resources/video_processing/hat_tracking.json" />
    <param name="dictionary" value="ARUCO"/>
    <param name="frame_rate" value="60" />
    <rosparam command="load" file="$(find hri_planner)/resources/video_processing/processor_setting.yaml" />
    <rosparam command="load" file="$(arg human_heights)" />
  </node>
</launch>
